test/e2e/upgradetest/upgrade_test.go

```diff
@@ -72,3 +72,67 @@ func TestResize(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+// TestBackupForOldCluster tests that new backup sidecar could make backup from old cluster.
```

Comment:
I'm wondering if we really need this.
Because there is already e2e test covering that the same version of backup could be used by same version of operator.